switch/readme-switch.md

```diff
@@ -0,0 +1,63 @@
+# Nintendo Switch Port of DevilutionX (Diablo)
+
+### How To Play:
+- Put `devilutionx.nro` in into `/switch/devilutionx`
+- Copy `DIABDAT.MPQ` from the original Diablo game disc or GOG version into `/switch/devilutionx`
+- Launch `devilutionx.nro` (do not use album to launch, see the note below)
+- *Note:* Hold R on any installed game and launch it. Do not use album to launch. If you use album, the homebrew only has very little memory available, and the touch keyboard doesn't work. This is true for all homebrew, not just Diablo-NX.
+
+### Joycon Controls
+
+- Left analog : move hero
```

Comment:
This is a bit out-of-date. Left analog click is now map. Quest log is Minus + Y. A/B and X/Y are swapped (better for the Switch actually).